Mathlib/Order/ReltoGaloisConnection.lean

```diff
@@ -67,25 +67,24 @@ theorem galoisConnection : GaloisConnection R.leftDual R.rightDual := by
 /-- `lFixedPoints` is the set of elements `J : Set α` satisfying `rightDual (leftDual J) = J`. -/
 def lFixedPoints := {J : Set α | R.rightDual (R.leftDual J) = J}
 
-/-- `rFixedPoints` is the set of elements `I : (Set β)ᵒᵈ` satisfying `leftDual (rightDual I) = I`.
--/
-def rFixedPoints := {I : (Set β)ᵒᵈ | R.leftDual (R.rightDual I) = I}
+/-- `rFixedPoints` is the set of elements `I : Set β` satisfying `leftDual (rightDual I) = I`. -/
+def rFixedPoints := {I : Set β | R.leftDual (R.rightDual I) = I}
 
 open GaloisConnection
 
 /-- `leftDual` maps every element `J` to `rFixedPoints`. -/
 theorem leftDual_mem_rFixedPoint (J : Set α) : R.leftDual J ∈ R.rFixedPoints := by
   unfold rFixedPoints
   apply le_antisymm
-  · exact R.galoisConnection.l_u_le (R.leftDual J)
-  · apply R.galoisConnection.monotone_l; exact R.galoisConnection.le_u_l J
+  · apply R.gc_leftDual_rightDual.monotone_l; exact R.gc_leftDual_rightDual.le_u_l J
+  · exact R.gc_leftDual_rightDual.l_u_le (R.leftDual J)
 
 /-- `rightDual` maps every element `I` to `lFixedPoints`. -/
 theorem rightDual_mem_lFixedPoint (I : (Set β)ᵒᵈ) : R.rightDual I ∈ R.lFixedPoints := by
```

Comment:
Should this mention of the dual order `(Set β)ᵒᵈ` also been removed?

Reply:
Yep, well spotted!

Reply:
Deleted. I'm happy with the changes. Thanks a lot Yael for your time and effort!